manifests/v1beta1/components/db-manager/db-manager.yaml

```diff
@@ -23,12 +23,9 @@ spec:
           image: docker.io/kubeflowkatib/katib-db-manager
```

Comment:
Since I need new image with this pr's modification, many e2e test failed.

Reply:
How could we test based on the change? Or maybe we just merge and fix if the tests fail in master.

Reply:
Actually you need to build a new image of db-manager based on this pr's branch.
Or I could make my image repository to public so that any reviewer could test with it easier.
Which one would be better?

Reply:
Both works for me. I will test it with my own build.

Reply:
Thanks for your effort :+1:

Reply:
I've tested with following kustomization configuration and following public docker image:
- https://github.com/anencore94/katib/tree/test-kustomization-for-postgres-db